examples_v20x/exti_blink/exti_blink.c

```diff
@@ -0,0 +1,60 @@
+#include "ch32fun.h"
+
+// specify interrupt line
```

Comment:
Please add a comment at the top of this file explaining what the test does and what IOs it's using for what.

Reply:
I have added this to the top. 

```
/*
Short example to demonstrate how to use external interrupts.
When I tested my circuit I made the following conections:
PA10 ---- LED
PA1  ---- Debounced switch that pulls to ground and a pull up resistor.
*/
```
Note though I have not committed it yet. Planning to re-read it later.

Reply:
Looks good.